examples/smbexec.py

```diff
@@ -265,7 +265,7 @@ def output_callback(data):
             self.transferClient.deleteFile(self.__share, OUTPUT_FILENAME)
         else:
             fd = open(SMBSERVER_DIR + '/' + OUTPUT_FILENAME,'r')
```

Comment:
```suggestion
            fd = open(SMBSERVER_DIR + '/' + OUTPUT_FILENAME,'rb')
```
This will ensure that reading from fd will return bytes, so no need to use encode, it is also compatible with python2